markdown/usage/troubleshooting.md

```diff
@@ -3,33 +3,88 @@ title: Troubleshooting
+
+If you are using a sample sheet or TSV input method, check there is not a mistake or typo in the path in a given column. Common mistakes ar e a trailing space at the end of the path, which can cause problems.
+
+### Output for only a single sample although I specified multiple with wildcards
+
+You must specify paths to files in quotes, otherwise your _shell_ (e.g. bash) will evaluate any wildcards (\*) rather than Nextflow.
+
+For example:
 
 ```bash
-ERROR ~ Cannot find any reads matching: *{1,2}.fastq.gz
+nextflow run nf-core/eager --input /path/to/sample_*/*.fq.gz
+```
+
+Maybe evaluated by your shell as:
+
+```bash
+nextflow run nf-core/eager --input /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz
+```
+
+And Nextflow will only take the first path after `--input`, ignoring the others.
+
+On the other hand, encapsulating the path in quotes will allow _Nextflow_ to evaluate the paths.
+
+```bash
+nextflow run nf-core/eager --input "/path/to/sample_*/*.fq.gz"
 ```
 
-Note that if your sample name is "messy" then you have to be very particular with your glob specification. A file name like `L1-1-D-2h_S1_L002_R1_001.fastq.gz` can be difficult enough for a human to read. Specifying `*{1,2}*.gz` wont work give you what you want Whilst `*{R1,R2}*.gz` will.
+### Data organization
 
-## Data organization
 The pipeline can't take a list of multiple input files - it takes a glob expression. If your input files are scattered in different paths then we recommend that you generate a directory with symlinked files. If running in paired end mode please make sure that your files are sensibly named so that they can be properly paired. See the previous point.
 
+## The pipeline crashes almost immediately with an early pipeline step
+
+Sometimes a newly downloaded and set up nf-core pipeline will encounter an issue where a run almost immediately crashes (e.g. at `fastqc`, `output_documentation` etc.) saying the tool could not be found or similar.
+
+### I am running Docker
+
+You may have an outdated container. This happens more often when running on the `dev` branch of a nf-core pipeline, because Docker will _not_ update the container on each new commit, and thus may not get new tools called within the pipeline code.
+
+To fix, just re-pull the pipeline's Docker container manually with:
+
+```bash
+docker pull nfcore/<PIPELINE>:dev
+```
+
+### I am running Singularity
+
+If you're running Singularity, it could be that Nextflow cannot access your Singularity image properly - often due to missing bind paths.
+
+See
+[here](https://nf-co.re/usage/troubleshooting#cannot-find-input-files-when-using-singularity)
+for more information.
+
 ## Image cannot be built
```

Comment:
```suggestion
## Singularity image cannot be built
```

Reply:
Would be good to move this and the next heading into a `## Singularity` section.